rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -40,7 +43,9 @@ class HVAC_SYS:
     SYS_12B = "Sys-12b"
     SYS_12C = "Sys-12c"
     SYS_13 = "Sys-13"
-    SYS_13A = "Sys-13A"
+    SYS_13A = "Sys-13a"
+    SYS_13B = "Sys-13b"
+    SYS_13C = "Sys-13c"
```

Comment:
No SYS_13B and C (see function is_baseline_system_13)

Reply:
When I saw the [22-1 RDS](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section22/Rule22-1.md#:~:text=11C%22%2C%20%22SYS%2D12C%22%2C-,%22SYS%2D13C%22,-%5D)%3A%20CHECK_RULE_LOGIC), there is  ```SYS-13C```. Should I remove this in the RDS?

Reply:
In the [is_baseline_system_13 RDS](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/is_baseline_system_13.md), there are only ```Sys-13``` and ```Sys-13a```. 

However, in some of section 22 RDS (e.g., [22-1](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section22/Rule22-1.md#:~:text=11C%22%2C%20%22SYS%2D12C%22%2C-,%22SYS%2D13C%22,-%5D))), there is"SYS-13C".

Is it correct to follow the is_baseline_system_13 RDS and fix section 22 RDS?